client/src/commons/Toggle/Toggle.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { ToggleButtonGroup, ToggleButton, ToggleButtonGroupProps } from '@material-ui/lab';
+
+import { useStyles } from './ToggleStyles';
+import theme from 'styles/theme';
+
+const Toggle: React.FC<Props> = (props: Props): JSX.Element => {
+    const classes = useStyles({});
+
+    const { firstOption, secondOption, value, ...rest } = props;
+    const activeButtonStyle = {
+        backgroundColor: theme.palette.primary.main, 
+        color: 'white'
+    }
+
+    return (
+        <ToggleButtonGroup value={value} exclusive {...rest}>
+            <ToggleButton className={classes.toggle} 
+                style={!value ? activeButtonStyle : {}}
+                value={false}>
+                {firstOption ? firstOption : 'לא'}
+            </ToggleButton>
+            <ToggleButton className={classes.toggle}
+                style={value ? activeButtonStyle : {}}
```

Comment:
why not export it to a class and add it conditionally in the classname

Reply:
Because then you must use important in order to override the current style, and we prefer style over important